### ISSUE TITLE
require-description-complete-sentence misreports line numbers for params

### ISSUE THREAD

**Issue by kewisch** [gajus/eslint-plugin-jsdoc#160]
`require-description-complete-sentence` reports the line number for the start of the comment, instead of the line where the period is missing.

```javascript
/**
 * Thing.
 *
 * @param {string} foo    A parameter
 */
function bar(foo) {
}
```

I'd expect it to flag line 4, but it flags line 1 for `Sentence must end with a period`.
